OpenRA.Game/Graphics/Vertex.cs

```diff
@@ -15,24 +15,19 @@ namespace OpenRA.Graphics
 	[StructLayout(LayoutKind.Sequential)]
 	public struct Vertex
 	{
-		// TODO Workaround for unused field warnings in mono 2.10
-		#pragma warning disable 414
-		float x, y, z, u, v;
-		float p, c;
-		#pragma warning restore
+		public readonly float X, Y, Z, U, V, P, C;
```

Comment:
We'll likely need to grow an 8th float when somebody gets around to doing the TS lighting stuff.